nilearn/signal.py

```diff
@@ -436,8 +442,8 @@ def clean(signals, sessions=None, detrend=True, standardize='zscore',
         Timeseries. Must have shape (instant number, features number).
         This array is not modified.
 
-    sessions : numpy array, optional
-        Add a session level to the cleaning process. Each session will be
+    runs : numpy array, optional
```

Comment:
Here we are changing the API. This needs a deprecation cycle (raise a warning, saying that sessions will result in an error after release 0.9)

Reply:
Yes this has been done in L414 with `rename_parameters`! I will add the description to the docstring as well.